### PR TITLE
OpenVX Harris corners wrappers

### PR DESCRIPTION
**Pull request by savuor**
**Disabled due to bad accuracy and multiple differences in implementation**

### CODE REVIEW

File: modules/imgproc/src/featureselect.cpp [hunk 1]
```diff
@@ -262,6 +268,117 @@ static bool ocl_goodFeaturesToTrack( InputArray _image, OutputArray _corners,
...
+                                         VX_MEMORY_TYPE_HOST, 0));
+#endif
+        std::vector<vx_keypoint_t> vxKeypoints(nPoints);
+        for(size_t i = 0; i < nPoints; i++)
+        {
+            vxKeypoints[i] = vxArrayItem(vx_keypoint_t, arrayPtr, i, arrayStride);
+        }
+
+#ifndef VX_VERSION_1_1
+        IVX_CHECK_STATUS(vxCommitArrayRange(corners, 0, nPoints, &arrayPtr));
```
Comment (jet47) [inline modules/imgproc/src/featureselect.cpp]: It should be `arrayPtr` not `&arrayPtr`.

File: modules/imgproc/src/featureselect.cpp [hunk 2]
```diff
@@ -262,6 +264,96 @@ static bool ocl_goodFeaturesToTrack( InputArray _image, OutputArray _corners,
...
+                                                 Image::createAddressing(image), image.data);
+        //The minimum threshold which to eliminate Harris Corner scores (computed using the normalized Sobel kernel).
+        //set to 0, we'll filter it later by threshold
+        ivx::Scalar strengthThresh = ivx::Scalar::create<VX_TYPE_FLOAT32>(context, 0);
+
+        //The gradient window size to use on the input.
+        ivx::Scalar gradientSize = ivx::Scalar::create<VX_TYPE_INT32>(context, 3);
+
+        //The block window size used to compute the harris corner score
+        ivx::Scalar blockSize = ivx::Scalar::create<VX_TYPE_INT32>(context, _blockSize);
```
Comment (apavlenko) [inline modules/imgproc/src/featureselect.cpp]: why do you create `blockSize` and `gradientSize` if you use `vx_int32` values only?

Comment (savuor) [inline modules/imgproc/src/featureselect.cpp]: Sure, forgot to remove this

File: modules/imgproc/src/featureselect.cpp [hunk 3]
```diff
@@ -262,6 +264,96 @@ static bool ocl_goodFeaturesToTrack( InputArray _image, OutputArray _corners,
...
+
+#ifdef VX_VERSION_1_1
+        //we should take user memory back before release
+        //(it's not done automatically according to standard)
+        ovxImage.swapHandle();
+#endif
+    }
+    catch (RuntimeError & e)
+    {
+        VX_DbgThrow(e.what());
```
Comment (apavlenko) [inline modules/imgproc/src/featureselect.cpp]: IMHO
```cpp
CV_DbgAssert(!"openvx_canny - WrapperError");
return false;
```
were much better!

**Comment by apavlenko** [comment]
:+1:

**Pull request merged**